src/tracing/node_trace_writer.cc

```diff
@@ -118,7 +124,7 @@ void NodeTraceWriter::FlushPrivate() {
   int highest_request_id;
   {
     Mutex::ScopedLock stream_scoped_lock(stream_mutex_);
-    if (total_traces_ >= kTracesPerFile) {
+    if (min_traces_per_file_ > 0 && total_traces_ >= min_traces_per_file_) {
```

Comment:
If the type is converted to a `uint64_t`, you can change the first test to be a `!=`.